src/patternfly/components/CodeEditor/code-editor-action-group-control.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-code-editor__action-group-control{{#if code-editor-action-group-control--modifier}} {{code-editor-action-group-control--modifier}}{{/if}}"
```

Comment:
what does this element do?

Reply:
It's a container around the overflow kebab element - I thought we should have a container in case we style the element later on. I renamed it `pf-c-code-editor__action-group-overflow`. wdyt?

Reply:
Oh so it's just a dropdown for overflow items, not necessarily an overflow menu component where you'll have items that collapse into the menu at a breakpoint?